tests/foreman/api/test_operatingsystem.py

```diff
@@ -165,6 +138,7 @@ def test_positive_end_to_end_os(self, module_target_sat, module_org, module_arch
                 'provisioning_template',
                 'ptable',
                 'medium',
+                'os_parameters_attributes',
```

Comment:
I think we're missing assertions to verify if new_os_parameters are updated?

Reply:
Updated